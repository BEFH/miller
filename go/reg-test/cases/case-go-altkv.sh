echo 'a,b,c,d,e,f'   | run_mlr --inidx --ifs comma altkv
echo 'a,b,c,d,e,f,g' | run_mlr --inidx --ifs comma altkv
