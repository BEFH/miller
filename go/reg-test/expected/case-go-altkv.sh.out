
mlr --inidx --ifs comma altkv


a=b,c=d,e=f

mlr --inidx --ifs comma altkv


a=b,c=d,e=f,4=g

